ldk/javascript/examples/self-test-loop/src/tests/whisper/text.ts

```diff
@@ -0,0 +1,102 @@
+import { stripIndent } from 'common-tags';
```

Comment:
@RusGraf  Do you think this is a good place to store those text? Do you think the file name and property names are Okay?

Reply:
I think it is a good start. We could could adapt the pattern on the go.